package.json

```diff
@@ -3,6 +3,33 @@
   "version": "0.13.23",
   "description": "Fast math typesetting for the web.",
   "main": "dist/katex.js",
+  "exports": {
+    ".": {
+      "require": "./dist/katex.js",
+      "import": "./dist/katex.mjs"
+    },
+    "./contrib/auto-render": {
+      "require": "./dist/contrib/auto-render.js",
+      "import": "./dist/contrib/auto-render.mjs"
+    },
+    "./contrib/mhchem": {
+      "require": "./dist/contrib/mhchem.js",
+      "import": "./dist/contrib/mhchem.mjs"
+    },
+    "./contrib/copy-tex": {
+      "require": "./dist/contrib/copy-tex.js",
+      "import": "./dist/contrib/copy-tex.mjs"
+    },
+    "./contrib/mathtex-script-type": {
+      "require": "./dist/contrib/mathtex-script-type.js",
+      "import": "./dist/contrib/mathtex-script-type.mjs"
+    },
+    "./contrib/render-a11y-string": {
+      "require": "./dist/contrib/render-a11y-string.js",
+      "import": "./dist/contrib/render-a11y-string.mjs"
+    },
+    "./*": "./*"
```

Comment:
Could you clarify that/why this is a breaking change?  I just tried installing this branch into a project (via `npm link`) and I could successfully do `require('katex/dist/contrib/auto-render.js')` as well as `require('katex/dist/contrib/auto-render')`.  Perhaps the `"./*"` rule is recursive, so it enables all old paths to still work?

If this doesn't actually work, perhaps it's worth adding rules like `"./dist/*"`, `"./dist/contrib/*"`, `"./src/*"` for backward compatibility?

Reply:
I just noticed `require('katex/contrib/auto-render')` wouldn't work before as well. The only breaking change is that `import katex from 'katex'` will now import the ECMAScript module (`dist/katex.mjs`), where it imported the CommonJS module (`dist/katex.js`) before, if supported. It'd not affect normal use, as module loaders/bundlers are smart enough to handle them, but the ECMAScript module uses newer syntax and it's technically a breaking change.

Reply:
Ah, right, that makes sense. I can't imagine what that would break, but it's certainly different behavior.

I wonder if it still makes sense to mention the new ways to import modules in `docs/migration.md` like you had before, but described as features instead of breaking changes?

Reply:
I think the current wording sounds like a feature:
```
With module loaders that support conditional exports and ECMAScript modules,
`import katex from 'katex';` will import the ECMAScript module.
```
WDYT?

Reply:
I agree; that text is good. I was referring to the following text, and wondering whether some of it should be included.  Maybe with a prefix like "As an alternative to importing `.js` or `.mjs` files directly, you can now..."

https://github.com/KaTeX/KaTeX/pull/3377/commits/25ff5b6f5cbfee86ca2afe00f72fef6ee917c471#diff-758ba107a728fccfdf0d098b2c06ded59127dd53ea287d630f72fb51040bc72dR34-R53

Reply:
I've added a table.